### PR TITLE
make URL canonical instead of PDF on paper page (closes #587)

### LINKED ISSUE #587: Make "URL" the canonical URL instead of .pdf
See #576 for a related issue.
I saw that the URL field on the paper sites point to the PDF.  I would assume that the paper pages show the same metadata as the bibtex files as they are meant as bibliographic information which someone might copy.  Therefore (and because the PDF is linked with the PDF button) I propose to use the canonical URL.

### PR DESCRIPTION
**Pull request by mjpost**

### CODE REVIEW

**Comment by akoehn** [comment]
This does not do the intended thing when compiling it on my machine, the result is this:
> URL: Page(/papers/P/P19/P19-1001.md)

Does it work correctly on your machines?

File: hugo/layouts/papers/single.html [hunk 1]
```diff
@@ -144,9 +144,13 @@ <h5 class="card-title">Abstract</h5>
       <dt>Pages:</dt>
       <dd>{{ with $paper.pages }}{{ . }}{{ end }}</dd>
       <dt>URL:</dt>
-      <dd>{{ with $paper.pdf }}<a href="{{ . }}">{{ . }}</a>{{ end }}</dd>
+      <dd><a href="{{ $paper.url }}">{{ . }}</a></dd>
```
Comment (mbollmann) [inline hugo/layouts/papers/single.html]: That second dot also needs to be replaced by `$paper.url` in this case. :)

Thanks @akoehn for testing & noticing!

**Comment by akoehn** [comment]
Looks good to me now, but why did you put an additional border between DOI and PDF?
border
I see no relevant difference between the PDF and the other fields, so I would simply skip that border.

**Comment by mjpost** [comment]
Yeah, it looks ugly. I was trying to separate data that's in the BibTex from other information about the paper. What about removing the border and putting the PDF line below the export formats line?

**Comment by aryamccarthy** [comment]
I'd just keep it all together. The SIG info isn't in the BibTeX either.

**Comment by akoehn** [comment]
agree with @aryamccarthy, this is an information block and not a bibtex information block.  The volume ID is also not part of the bibtex. 

Adding a line primarily makes people like me second-guess what the reason for the line could be :-)

**Comment by mjpost** [comment]
Okay, I restored it. I moved the PDF link to the bottom because it's already extraneous and it looked ugly next to the URL line (since they're nearly identical in most cases).

**Comment by mjpost** [comment]
Feel free to merge this if no one complains about me having moved it...:)

**Comment by akoehn** [comment]
@mbollmann you requested changes therefore you need to approve before it can be merged.

**Pull request merged**